packages/browserslist-config-terra/CHANGELOG.md

```diff
@@ -1,45 +1,49 @@
-ChangeLog
-=========
+# Changelog
 
-Unreleased
------------------
+## Unreleased
+
+## 3.0.0 - (May 5, 2020)
+
+Breaking Change
 
-3.0.0 - (May 5, 2020)
-------------------
-### Breaking Changes
 * Updated to a scoped packages: @cerner/browserslist-config-terra
 
-2.0.0 - (January 28, 2020)
-------------------
-### Breaking
+## 2.0.0 - (January 28, 2020)
+
+Breaking
```

Comment:
I had a question [here](https://github.com/cerner/terra-core/pull/3091#discussion_r463108827) about these headings. Do we want these to be unformatted text or triple #'s? 


```
Changed
```

```
### Changed
````

Reply:
You don't want headings to repeat.. so multiple `### Changed` headings in a document will break anchor links. Also this feedback would have been more helpful 80 files ago 😝

Reply:
i checked terra ui and actually we're smart in our md processing and give unique tags for repeated headings... still I can't say i'm excited to go and update all the change logs again... i think this is fine???

Reply:
The Github markdown compiler seems to be smart enough to append numeric indexing on multiple occurrences of the same heading.

https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/CHANGELOG.md#added

https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/CHANGELOG.md#added-1

Are the duplicate headings a warning on the linter? This is kind of an unfortunate situation for the change logs that rely on repeat headings

Reply:
Since this is going to be the standard we enforce I'd say we go with what we believe is the right way to do it. Whichever of the two we decide

Reply:
I just redeployed terra-ui, heres what this format looks like: https://engineering.cerner.com/terra-ui/components/terra-arrange/arrange/change-log

Reply:
We landed on using lists:
```md
* Changed
  * Thing that changed
```